tmt/steps/discover/shell.py

```diff
@@ -42,32 +46,34 @@ class DiscoverShell(tmt.steps.discover.DiscoverPlugin):
     # Supported methods
     _methods = [tmt.steps.Method(name='shell', doc=__doc__, order=50)]
 
-    def show(self, keys=None):
+    def show(self, keys: Optional[List[str]] = None) -> None:
         """ Show config details """
         super().show([])
         tests = self.get('tests')
         if tests:
             test_names = [test['name'] for test in tests]
             click.echo(tmt.utils.format('tests', test_names))
 
-    def wake(self, keys=None):
+    def wake(self, keys: Optional[List[str]] = None) -> None:
         """ Wake up the plugin, process data, apply options """
         super().wake(keys=keys)
         # Check provided tests, default to an empty list
         if 'tests' not in self.data:
             self.data['tests'] = []
-        self._tests = []
+        self._tests: List[tmt.base.Test] = []
 
-    def go(self):
+    def go(self, *args: Any, **kwargs: Any) -> None:
```

Comment:
Are `*args` and `**kwargs` needed? `DiscoverShell` inherits from `DiscoverPlugin` whose `def go(self) -> None` accepts no arguments anyway. (See https://github.com/teemtee/tmt/commit/b371a0d0d04c3e16384f3f1749f5b08efb2c7097 for the related refactoring).

Reply:
Right. This is something I forgot to do after the `go()` refactoring. Fixed, thanks.